amr-wind/equation_systems/sdr/SDR.H

```diff
@@ -44,6 +45,46 @@ struct SDR : ScalarTransport
     static constexpr bool need_nph_state = true;
 };
 
+template<>
+struct TurbulenceOp<SDR>
```

Comment:
It would be good to consolidate all Ops in `sdr_ops` files

Reply:
https://github.com/Exawind/amr-wind/pull/322/commits/cac0cf80a0d3db19f2afc6164c91424bf91902b7